tools/classes.js

```diff
@@ -150,6 +150,18 @@ var classes = [
   'foam.box.socket.SocketClientReplyBox',
   'foam.box.socket.SocketServer',
   'foam.box.socket.SslContextFactory',
+  'foam.box.sf.SFBOX',
```

Comment:
Believe these now need to be added to files.js with flags [ 'java' ] now as well.